### PR TITLE
⚡️ PIC-1355: Constrain default case list query

### PR DESCRIPTION
**Pull request by jhevans**
Constrain default case list query to cases created 8 days before hearing date

### CODE REVIEW

File: src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java [hunk 1]
```diff
@@ -48,14 +48,14 @@
...
     @ApiResponses(
-        value = {
-            @ApiResponse(code = 200, message = "OK", response = CourtCaseResponse.class),
-            @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
-            @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class),
-            @ApiResponse(code = 403, message = "Forbidden", response = ErrorResponse.class),
-            @ApiResponse(code = 404, message = "Not Found. For example if the court code or case number can't be matched.", response = ErrorResponse.class),
-            @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
-        })
+            value = {
```
Comment (jhevans) [inline src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java:51]: Gah, sorry - auto formatted

File: src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java [hunk 2]
```diff
@@ -109,16 +109,18 @@ CaseListResponse getCaseList(
 
     ) {
         final var createdAfterOrDefault = Optional.ofNullable(createdAfter)
-                .orElse(LocalDateTime.of(LocalDate.EPOCH, LocalTime.MIDNIGHT));
+                .orElse(
```
Comment (jhevans) [inline src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java:112]: This is the change

File: src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java (hunk 1 above)
Comment (chris-faulkner-mojdt) [reply src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java:51]: NP

**Review by chris-faulkner-mojdt — APPROVED** [review]
LGTM

**Pull request merged**